UNITTESTS/stubs/ATHandler_stub.cpp

```diff
@@ -43,23 +43,34 @@ bool ATHandler_stub::bool_value = false;
 uint8_t ATHandler_stub::uint8_value = 0;
 FileHandle_stub *ATHandler_stub::fh_value = NULL;
 device_err_t ATHandler_stub::device_err_value;
-Callback<void()> ATHandler_stub::callback = NULL;
```

Comment:
I believe this breaks unittests, see failure here https://mbed-os.mbedcloudtesting.com/blue/organizations/jenkins/mbed-os-ci_unittests/detail/mbed-os-ci_unittests/1295/pipeline (internal link)